WWW/2_linear_chain.wdl

```diff
@@ -54,24 +54,25 @@ workflow minidata_test_alignment {
       ref_sa = ref_sa,
       taskDocker = bwadocker
   }
-  # Outputs that will be retained when execution is complete
-  output {
-    File analysisReadyBam = "~{base_file_name}" +".aligned.bam"
-    File analysisReadyBamSorted = "~{base_file_name}" +"_sorted_query_aligned.bam"
-  }
    
-   # Mark duplicates
-  call MarkDuplicatesSpark as MarkDuplicates {
+  # Mark duplicates
+  call MarkDuplicatesSpark {
     input:
-      input_bam = ~{base_file_name} + "_sorted_query_aligned.bam",
-      output_bam_basename = base_file_name + ".aligned.duplicates_marked",
-      metrics_filename = base_file_name + ".duplicate_metrics",
+      input_bam = BwaMem.analysisReadySorted,
+      output_bam_basename = "~{base_file_name}.duplicates_marked",
+      metrics_filename = "~{base_file_name}.duplicate_metrics",
       taskDocker = GATKdocker
   }
+
+  # Outputs that will be retained when execution is complete
+  output {
+    File alignedBamSorted = BwaMem.analysisReadySorted
+    File markDuplicates = MarkDuplicatesSpark.output_bam
+  }
 # End workflow
 }
-#### TASK DEFINITIONS
 
+#### TASK DEFINITIONS
 # align to genome
 ## Currently uses -M but GATK uses -Y and no -M
 task BwaMem {
```

Comment:
Because ref_alt, ref_amb, ref_ann, ref_bwt, ref_pac, and ref_sa are all unused in the BwaMem task, there's no reason to pass them into the task. These should also be removed from the workflow level arguments circa line 27.